features/openhab-addons/pom.xml

```diff
@@ -35,25 +35,25 @@
         <executions>
           <execution>
             <id>create-karaf-features</id>
-            <phase>generate-sources</phase>
             <goals>
               <goal>run</goal>
             </goals>
+            <phase>generate-sources</phase>
             <configuration>
               <target>
-                <concat destfile="src/main/feature/feature.xml"> 
-                  <header file="src/main/resources/header.xml" filtering="no"/>
+                <concat destfile="src/main/feature/feature.xml">
+                  <header file="src/main/resources/header.xml" filtering="no"></header>
                   <fileset dir="${basedirRoot}/bundles">
-                    <include name="*/src/main/feature/feature.xml"/>
-                    <exclude name="**/org.openhab.binding.bluetooth*/**/feature.xml"/>
-                    <exclude name="**/org.openhab.binding.mqtt*/**/feature.xml"/>
+                    <include name="*/src/main/feature/feature.xml"></include>
+                    <exclude name="**/org.openhab.binding.bluetooth*/**/feature.xml"></exclude>
+                    <exclude name="**/org.openhab.binding.mqtt*/**/feature.xml"></exclude>
                   </fileset>
                   <filterchain>
                     <linecontainsRegExp>
-                      <regexp pattern="(feature>)|(feature\s)|(bundle>)|(bundle\s)"/>
+                      <regexp pattern="(feature&gt;)|(feature\s)|(bundle&gt;)|(bundle\s)"></regexp>
```

Comment:
Did you check whether this has no unexpected side effects?

Reply:
No. Didn’t see that. Came out of the automatic formatter. Will check the resulting feature-file.

Reply:
md5-hash is the same for both. so we can keep the new one